twolde.py

```diff
@@ -205,11 +207,18 @@ def nice_time(seconds):
 
 def main():
     """ main """
-    parser = argparse.ArgumentParser(description='')
+    parser = argparse.ArgumentParser(
+        description="It's like Twitshift, but for people who prefer Twolde",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
     parser.add_argument(
         dest='command',
         choices=['install', 'run', 'remove'],
         help='the twolde command to run')
+    parser.add_argument(
```

Comment:
Yep, but because `install` and `remove` don't take anything else, perhaps it's simpler just to silently ignore `--years` for those?